_haikus/bunny.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: haiku
+title: bunny is fat
+author: sciduck
+---
+my bunny is fat
+he loves to eat cabbage
```

Comment:
This line needs an extra syllable in addition to a line break. A possible alternative:

```
he does love to eat cabbage <br>
```